### DISCUSSION TITLE
Create bounds passed on flightPath

### DISCUSSION THREAD

**Discussion by dsumah** [inocan-group/vue3-google-map#229] [Q&A]
Hi guys. How do I go about setting the correct bounds for my map? What I am trying to achieve is a situation where my polyline fits or almost fits the screen.

It wil be nice if someone can point to where the docs speaks about setting bounds

<img width="511" alt="Capture d’écran 2022-10-27 à 12 25 29 AM" src="https://user-images.githubusercontent.com/106399477/198157198-b4d6fd39-e685-414a-ada9-f654570ff415.png">

**Comment by Mastersam07** [comment]
Not sure this is supported out of the box. You will have to use the native api to fit bounds. You could check the [advanced usage](https://github.com/inocan-group/vue3-google-map#advanced-usage) section of the docs.

```
var bounds = new window.google.maps.LatLngBounds()
bounds.extend(pathA)
bounds.extend(pathB)
...
bounds.extend(pathN)
            
this.$mapRef.value.map.fitBounds(bounds)
```

in the above, mapRef is the reference to your google map(see [advanced usage](https://github.com/inocan-group/vue3-google-map#advanced-usage))

If you have a list of LatLng values, you could loop over and extend the bounds for each of the value in thelist.
